File: program/print-numbers-from-1-to-n-without-using-a-loop/print-numbers-from-1-to-n-without-using-a-loop.kt
fun main() {
  print("Enter number : ")
  println()
  var input = readLine()
  printNumber(input!!.toInt(), input!!.toInt()-1)
}

fun printNumber(originalNum: Int, num: Int): Int {
  println(originalNum-num)
  if (num == 0) {
   return originalNum
  }
  return printNumber(originalNum, num-1)
}

File: program/print-numbers-from-1-to-n-without-using-a-loop/PrintNumbersFrom1ToNWithoutUsingALoop.swift
import Swift
import Foundation

func printNumber(original:Int, n: Int) -> Int {
   print(original - n)
   if(n == 0) {
     return 0
   }
   return printNumber(original: original, n: n-1)
}

var num = 15
var result1 = printNumber(original: num, n: num-1)
